backend/apps/web/routers/auths.py

```diff
@@ -104,8 +104,8 @@ async def signup(form_data: SignupForm):
                     "profile_image_url": user.profile_image_url,
                 }
             else:
-                raise HTTPException(500, detail=ERROR_MESSAGES.DEFAULT(err))
+                raise HTTPException(500, detail=ERROR_MESSAGES.DEFAULT())
```

Comment:
suggestion: I'd log the `err` , might be useful for debugging.

Reply:
Sounds good!